src/AtomShell/install.jl

```diff
@@ -14,28 +14,28 @@ function install()
 
     if is_apple()
       rm′("Julia.app")
-      shell = "electron-v$version-darwin-x64.zip"
-      download("https://junolab.s3.amazonaws.com/blink/$shell")
-      run(`unzip -q $shell`)
-      rm(shell)
+      file = "electron-v$version-darwin-x64.zip"
+      download("https://github.com/electron/electron/releases/download/v$version/$file")
```

Comment:
This change is why it's no longer working on mac! Before your change, this code used to download the following file:
https://junolab.s3.amazonaws.com/blink/electron-v0.35.2-darwin-x64.zip

This is hosted by JunoLabs, not electron, and it's a slightly modified version of the Electron.app that comes from downloading the file *you* now download:
https://github.com/electron/electron/releases/download/v1.6.15/electron-v1.6.15-darwin-x64.zip

As far as I can tell, the one hosted by JunoLabs, is just the Electron app renamed to `Julia`, and with the julia logo. The contents of the two apps are identical.

![screen shot 2017-12-23 at 3 04 12 am](https://user-images.githubusercontent.com/1582097/34318156-4eb8d776-e78e-11e7-8829-2489d3f5cfd2.png)
![screen shot 2017-12-23 at 3 04 23 am](https://user-images.githubusercontent.com/1582097/34318157-4ec56856-e78e-11e7-8631-51020fdcdcb8.png)


So anyway, the code here is currently expecting to find the Electron binary inside an app named "Julia.app":
https://github.com/JunoLab/Blink.jl/blob/20dfac1887cf396dbca8dfd6948cc6cc82ce9b3c/src/AtomShell/process.jl#L43

Reply:
So, either you can host a newer version of the Electron app renamed to Julia.app on AWS, and download that, *or* you can just change the code in `process.jl` to refer to "deps/Electron.app/Contents/MacOS/Electron".

I'd probably just change the code.

Reply:
(but of course, as expected, when you use the stock Electron.app, you wind up with the stock electron icon and name):

<img width="243" alt="screen shot 2017-12-23 at 3 36 04 am" src="https://user-images.githubusercontent.com/1582097/34318297-71f3d62e-e792-11e7-8b41-3e267e6fe063.png">

So it might be worth it either to investigate AWS or making those same changes I made, but doing them programatically during the installation...